src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-new-relics-query-language.mdx

```diff
@@ -225,3 +225,5 @@ NRQL lets you query nearly every type of our telemetry data, including:
 <Callout variant="tip">
   Some data, like relationships between monitored entities, is not available via NRQL but is available using our [NerdGraph API](/docs/apis/nerdgraph/get-started/introduction-new-relic-nerdgraph).
 </Callout>
+
+Ready to learn more? Check out our docs on [how to use NRQL](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-how-nrql-works/) and [how to use charts and dashboards with NRQL](/docs/query-your-data/nrql-new-relic-query-language/get-started/charts-and-dashboards-with-nrql/). If you want to start using NRQL instead, jump straight into our [guided NRQL tutorial](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-tutorial/).
```

Comment:
Same with 'our docs'. I think I'd remove 'our docs', I think it's better 'Check out how to use....'

Reply:
Good suggestion, I will make a change. Thank you!